admin/class-collapsible-presenter.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Collapsible_Presenter.
+ */
+class WPSEO_Collapsible_Presenter extends WPSEO_Paper_Presenter {
+
+	/**
+	 * WPSEO_presenter_collapsible constructor.
+	 *
+	 * @param string $title     The title of the collapsible.
+	 * @param string $view_file Optional. The path to the view file. Use the content setting if you do not wish to use
+	 *                          a view file.
+	 * @param array  $settings  Optional. Settings for the collapsible.
+	 */
+	public function __construct( $title, $view_file = null, array $settings = [] ) {
```

Comment:
If `$view_file` must be a string, I'm not sure why the default should be `null`. I guess I'd use an empty string as default value.